### ISSUE TITLE
RequireOnlyStandaloneIncrementAndDecrementOperators: false positive

### ISSUE THREAD

**Issue by jrfnl** [slevomat/coding-standard#568]
The following test case results in a false positive:
```php
switch($foo) {
	case '1':
		$bar++;
		break;
}
```

```
------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------
 43 | ERROR | Post-increment operator should be used only as single instruction.
    |       | (SlevomatCodingStandard.Operators.RequireOnlyStandaloneIncrementAndDecrementOperators.
PostIncrementOperatorNotUsedStandalone)
------------------------------------------------------------------------------------------
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/3bc5abbcd1f1d50cf4b2c1dcbf4c2e7bea8747fc
